lib/button.js

```diff
@@ -126,7 +126,7 @@ function Button(opts) {
 
       timeout = setTimeout(function() {
         if (this.isDown) {
-          this.emit("hold", err);
+          this.emit("hold");
```

Comment:
Other examples of data-less events use date stamps.  Should we standardize on this?